packages/site-client/src/pages/import/wco-catalog-import-page.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {html} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import {WCOPage} from '../../shared/wco-page.js';
+import {Task} from '@lit-labs/task';
+
+@customElement('wco-catalog-import-page')
+export class WCOCatalogImportPage extends WCOPage {
+  private _importTask = new Task(this, {
+    task: async ([packageName]: [packageName: string | undefined]) => {
+      console.log('_importTask', packageName);
+
+      if (packageName !== undefined && packageName.trim().length > 0) {
+        const response = await fetch(`/catalog/import`, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            packageName,
+          }),
+        });
+        const result = await response.json();
```

Comment:
Think we need to check the response code before parsing the json? Since fetch doesn't throw on HTTP errors.

Reply:
I think we can let `.json()` throw and add a error template.